cfme/tests/infrastructure/test_ssa_vddk.py

```diff
@@ -47,13 +48,16 @@ def ssa_analysis_profile(appliance):
 @pytest.fixture(params=vddk_versions, ids=([item for item in vddk_versions]), scope='function')
 def configure_vddk(request, appliance, provider, vm):
     vddk_version = request.param
-    vddk_url = conf.cfme_data.get("basic_info").get("vddk_url").get(vddk_version)
+    vddk_url = conf.cfme_data.get("basic_info", {}).get("vddk_url", {}).get(vddk_version, None)
+    if vddk_url is None:
+        pytest.skip('Could not locate vddk url in cfme_data')
+    else:
+        appliance.install_vddk(vddk_url=vddk_url)
```

Comment:
Please confirm that its okay to try and install_vddk before host credentials are set? 5.11 is failing with:

```Task Scan from Vm test-ssa-qp0m error: no eligible proxies for vm :[[nfs_datastore_1] test-ssa-qp0m/test-ssa-qp0m.vmx] - [provide credentials for this vm's host to perform smartstate analysis], aborting job [d3aa434c-561c-483d-806c-cd472612c1eb].```

Reply:
@mshriver this is because of REST issue. @valaparthvi had raised BZ around it.
I will add UI work around till REST BZ fixes. 
P.S. REST BZ is affecting only 5.11 hosts credentials. PR #8869 also blocked for it.

Reply:
@mshriver added UI workaround for it.